lib/build.gradle.kts

```diff
@@ -141,70 +142,69 @@ publishing {
+                        email.set("jakub.ptak@swmansion.com")
+                    }
+                    developer {
+                        id.set("arturmichalek")
+                        name.set("Artur Michałek")
+                        email.set("artur.michalek@swmansion.com")
+                    }
+                    developer {
+                        id.set("bartoszrybarski")
+                        name.set("Bartosz Rybarski")
+                        email.set("bartosz.rybarski@swmansion.com")
+                    }
+                    developer {
+                        id.set("wojciechszymczyk")
+                        name.set("Wojciech Szymczyk")
+                        email.set("wojciech.szymczyk@swmansion.com")
+                    }
                 }
-                developer {
-                    id.set("bartoszrybarski")
-                    name.set("Bartosz Rybarski")
-                    email.set("bartosz.rybarski@swmansion.com")
+                scm {
+                    connection.set("scm:git:git://github.com/software-mansion/starknet-jvm.git")
+                    developerConnection.set("scm:git:ssh://github.com:software-mansion/starknet-jvm.git")
+                    url.set("https://github.com/software-mansion/starknet-jvm/tree/main")
                 }
-                developer {
-                    id.set("wojciechszymczyk")
-                    name.set("Wojciech Szymczyk")
-                    email.set("wojciech.szymczyk@swmansion.com")
+                pom.withXml {
+                    val dependencyNode = asNode().appendNode("dependencies")
+
+                    val kotlinDependency = dependencyNode.appendNode("dependency")
+                    kotlinDependency.appendNode("groupId", "org.jetbrains.kotlin")
+                    kotlinDependency.appendNode("artifactId", "kotlin-stdlib-jdk8")
+                    kotlinDependency.appendNode("version", "1.3.50")
+
+                    val bouncyCastleDependency = dependencyNode.appendNode("dependency")
+                    bouncyCastleDependency.appendNode("groupId", "org.bouncycastle")
+                    bouncyCastleDependency.appendNode("artifactId", "bcprov-jdk15on")
+                    bouncyCastleDependency.appendNode("version", "1.70")
+
+                    val okHttpDependency = dependencyNode.appendNode("dependency")
+                    okHttpDependency.appendNode("groupId", "com.squareup.okhttp3")
+                    okHttpDependency.appendNode("artifactId", "okhttp")
+                    okHttpDependency.appendNode("version", "4.10.0")
+
+                    val kotlinxDependency = dependencyNode.appendNode("dependency")
+                    kotlinxDependency.appendNode("groupId", "org.jetbrains.kotlinx")
+                    kotlinxDependency.appendNode("artifactId", "kotlinx-serialization-json")
+                    kotlinxDependency.appendNode("version", "1.3.3")
                 }
```

Comment:
How complicated it'd be to include these automatically based on main build.gradle.kts? That way we won't have to remember to update these every time we bump a dependency version.

Reply:
The problem is, we have mutliple deps decalred in the main build file, and not all of them should be noted as dependencies for the package (eg. test stuff). So, not sure how complex, but I imagine quite, as we still would have to somehow specify what goes in here one way or the other

Reply:
I see we already used hardcoded dependencies in the previous version. Could you open an issue for making these automatically included? For the time being, we can merge it as is.

Reply:
https://github.com/software-mansion/starknet-jvm/issues/143